src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampTokenInfoNet472Wrapper.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Security.Cryptography;
+
+namespace NuGet.Packaging.Signing
+{
+#if IS_SIGNING_SUPPORTED && IS_DESKTOP
+    internal class Rfc3161TimestampTokenInfoNet472Wrapper : IRfc3161TimestampTokenInfo
+    {
+        private Rfc3161TimestampTokenInfo _rfc3161TimestampTokenInfo;
+
+        public Rfc3161TimestampTokenInfoNet472Wrapper(byte[] timestampTokenInfo)
+        {
+            _rfc3161TimestampTokenInfo = new Rfc3161TimestampTokenInfo(timestampTokenInfo);
+        }
+
+        public Rfc3161TimestampTokenInfoNet472Wrapper(Rfc3161TimestampTokenInfo timestampTokenInfo)
+        {
+            _rfc3161TimestampTokenInfo = timestampTokenInfo;
+        }
+        public string PolicyId
```

Comment:
Add 1 blank line before this property.

Also, the verbosity of these read-only properties can be cut down by using expression body syntax:
```C#
public string PolicyId => _rfc3161TimestampTokenInfo.PolicyId;
```

This feedback applies globally (e.g.:  Rfc3161TimestampTokenInfoNetstandard21Wrapper.cs).

Reply:
Thanks! Fixed.

Reply:
What about other properties in this same file?  What about properties in Rfc3161TimestampTokenInfoNetstandard21Wrapper.cs?

Reply:
Check all your Rfc3161*Wrapper files.

Reply:
Thanks! Changed all readonly properties into expression body syntax in Rfc3161*Wrapper files.